.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- Hello! This template is automatically added to help write up your pull request. Feel free to delete these comments, although they won't show up in the rendered markdown! This is only a template, feel free to adjust as you please! -->
+
+## 🚀 Description
+
+<!-- Please write a thoughtful description of what this PR is accomplishing. Are you adding a new feature? Fixing a bug? Writing documentation? Is there a GitHub issue that can be closed if this merges? -->
+
+---
```

Comment:
Suggestion: I kind of like keeping a summary of changes in list form:

```suggestion
---
- [x] added: new styles in {filename}
- [x] fixed: addressed flicker in {filename}
```

Reply:
💥 Added! Thank you!